_data/schedules/25-summer.yml

```diff
@@ -10,4 +10,73 @@
     - time: "8:30"
       route_id: gas-works-locks-salmon-bay-loop
     - time: "10:00"
-      route_id: gas-works-lake-union-loop
\ No newline at end of file
+      route_id: gas-works-lake-union-loop
+
+- date: "2025-07-05"
+  plan:
+    - time: "8:30"
+      route_id: westlake-jack-perry-memorial-park-ob
+    - time: "9:30"
+      route_id: westlake-dravus-street-ob
+
+- date: "2025-07-12"
+  plan:
+    - time: "8:30"
+      route_id: locks-discovery-radar-magnolia-discovery-ob
+
+- date: "2025-07-19"
+  plan:
+    - time: "8:30"
+      route_id: northgate-cemeteries-interurban-boeing-creek-ob
+
+- date: "2025-07-26"
+  plan:
+    - time: "8:30"
+      route_id: cse-arb-lwb-seward-loop-cc
+
+- date: "2025-08-02"
```

Comment:
We had a fair amount of Westlake already in July... maybe we should consider shifting this one?